lerna.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "node_modules/lerna/schemas/lerna-schema.json",
   "useWorkspaces": true,
-  "version": "6.5.0-alpha.0",
+  "version": "independent",
```

Comment:
is this right?

Reply:
Yes, so that each package can have its own version, no need to release new versions of a package if changes weren't made to them IMO.